libraries/lib-sample-track/Mix.cpp

```diff
@@ -201,6 +205,45 @@ void Mixer::MakeResamplers()
       mResample[i] = std::make_unique<Resample>(mHighQuality, mMinFactor[i], mMaxFactor[i]);
 }
 
+bool Mixer::NeedsDither(const SampleTrackConstArray &inputTracks) const
+{
+   // Many possible disqualifiers for the avoidance of dither
+   if (mbVariableRates)
+      // We will call MixVariableRates(), so we need nontrivial resampling
+      return true;
+
+   for (const auto &pTrack : inputTracks) {
+      auto &track = *pTrack;
+      if (track.GetRate() != mRate)
+         // Also leads to MixVariableRates(), needs nontrivial resampling
+         return true;
```

Comment:
I have a doubt about this. 
I create two tracks, by default at 44.1k, then resample one of them at 48k, then mix the two tracks to one, so this "if" statement will be true and this method will say that dithering is needed. However, the two tracks have the same bit depth, only thing is that one of them will need to be resampled before mixing - I think dithering should not be applied in this case

Reply:
Alright then, dither is neeed in this case too.